gordo_components/data_provider/iroc_reader.py

```diff
@@ -59,14 +59,12 @@ def load_series(
             map(lambda tag: IrocReader.base_path_from_asset(tag.asset), tag_list)
         )
         if len(set(base_paths_from_assets)) > 1:
```

Comment:
Safer to check if the len != 1 perhaps? If the len of the set is 0 (zero) is that ok?

Reply:
For me, the case len = 0 does not exist because the base_path_from_asset will always return something (won't it?), so this check is only for the case where more than 1 asset is found. The next check is supposed to handle when no asset has been found for some tags.
I might be mistaking of course :)

Reply:
In a way yes - but if for example the tag list was empty, I guess base_path_from_asset would return an empty list? And the next test (testing equal lenghts) would also pass. 

In the code today, we have already tested that the tag list is not empty, but that is a bit risky to assume, someone might change the lines above. I don't see what allowing for a lenght==0 gives here, it seems to me that the most precise thing to test in this "if" is is if the set is exactly lenght==1.

Reply:
Understood, I was taking for granted that the rest of the checks would not be changed but the world is a nasty place, so this is a fair point :)